controllers/AdminController.php

```diff
@@ -59,7 +59,7 @@ public function actionCommonDrugs()
 	 * @param $ssd_id
 	 * @return html
 	 */
-	public function actionCommonDrugsDelete($ssdId)
+	public function actionCommonDrugsDelete($itemId)
```

Comment:
This has been changed to itemId by the javascript is still sending ssdId so delete is not working.